test/rules/no-non-arrow-functions/test.ts.lint

```diff
@@ -0,0 +1,23 @@
+function foo(a: any) : any { // error
```

Comment:
Don't need the `// error` comments in this file since the markup indicates that itself